### DISCUSSION TITLE
Events

### DISCUSSION THREAD

**Discussion by LoganKavalSchool** [LoganKaval/MulBX#147] [General]
Events Taking Place Involving This Repository

**Comment by LoganKavalSchool** [comment]
We Are Attempting to Get Hyprland and All the Necessary Applications Working!!! Some Applications include Hyprpaper and Waybar.

**Reply by LoganKaval** [reply]
Hyprland has been officially left behind and we are now focusing on River. There are a few issues that need fixed before we can post the finish conifg files.

**Reply by LoganKavalSchool** [reply]
A new discussion named "River Instead of XMonad" will now contain the rest of this news.

**Comment by LoganKaval** [comment]
We Are Renaming MulBX to be Jaiyk Linux!!! This change will be made in the 3.9.0 release.

**Comment by LoganKaval** [comment]
The Prerelease of Jaiyk v3.9.0 is Out!!!

**Comment by LoganKaval** [comment]
The Alpha Release of Jaiyk(MulBX) 3.9.0 Nehir is Now Out!!!! The Debian Version Still Uses XMonad Due to the Repos Not Containing The Needed Packages.
